roles/opensrp/defaults/main.yml

```diff
@@ -117,7 +117,17 @@ opensrp_cors_allowed_source: ""
 opensrp_cors_max_age: "60"   
 
 #Team module settings
-opensrp_use_opensrp_team_module: false  
+opensrp_use_opensrp_team_module: false
+
+# S3 configuration (should be populated for deployments using S3 multimedia storage)
+aws_access_key_id: ""
```

Comment:
@vincent-karuri please prefix all these variables with `opensrp_`

Reply:
@jasonrogena That's possible but it would require changing the code that depends on it in opensrp server

Reply:
Can this change be part of this issue #154?

This was just meant to be a quick fix to allow deployments

Reply:
The Ansible variables, and not the ones defined in the .properties file.

Reply:
Done.